### ISSUE TITLE
add mxnet support?

### ISSUE THREAD

**Issue by yupbank** [databricks/tensorframes#109]
well... from our experience, mxnet are more friendly with resources and the interface is more compact and clean.. is there a plan to support mxnet?

**Comment by thunterdb** [comment]
Yes, this is a good point. I have not talked too much with the MXNet people about that, but this would certainly be a neat feature in the future.

@yupbank , since you worked with both TF and MXNet, do you have some quantitative arguments for one or the other framework?

**Comment by yupbank** [comment]
hmm.. https://github.com/fchollet/keras/issues/1313  had some discussion for that
